tests/integration/test_stepfunctions.py

```diff
@@ -30,6 +30,50 @@
         }
     }
 }
+TEST_LAMBDA_NAME_3 = 'lambda_map_sfn_3'
+MAP_STATE_MACHINE_NAME = 'test_map_sm_3'
+MAP_STATE_MACHINE_DEF = {
+    'Comment': 'Hello Map State',
+    'StartAt': 'ExampleMapState',
+    'States': {
+        'ExampleMapState': {
+            'Type': 'Map',
+            'Iterator': {
+                'StartAt': 'CallLambda',
+                'States': {
+                    'CallLambda': {
+                        'Type': 'Task',
+                        'Resource': '__tbd__',
+                        'End': True
+                    }
+                }
+            },
+            'End': True
+        }
+    }
+}
+MAP_STATE_MACHINE_OUTPUT = [
```

Comment:
Nitpick: Not sure if we need to define the outputs/inputs as global variables here (could be moved into the test method to keep the test definition lean/short). Shouldn't hold up the merge, though - could be tackled in a follow-up PR.